src/main/java/core/basesyntax/service/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import core.basesyntax.FruitTransaction;
+
+public interface OperationHandler {
+    void operationHandler(FruitTransaction fruitTransaction);
```

Comment:
method names are verbs that represents what should be done or returned by it
```suggestion
    void processTransaction(FruitTransaction fruitTransaction);
```